cyctl/internal/update/template_store.go

```diff
@@ -0,0 +1,94 @@
+package update 
+
+import (
+	"fmt"
+
+	"github.com/cyclops-ui/cyclops/cyclops-ctrl/api/v1alpha1/client"
+	"github.com/cyclops-ui/cycops-cyctl/internal/kubeconfig"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	
+)
+
+var (
+	updateTemplateStoreExample = `# updates template store; takes template name as an argument with flag --value
+
+	# Update template sample command 
+	cyctl update template NAME --repo='https://github.com/cyclops-ui/templates' --path='/path' --version='main' --icon='icon'`
+)
+
+var (
+	repo string
+	path string
+	icon string
+	version string 
+)
+
+// updates he given template from cyclops API
+func updateTemplate(clientset *client.CyclopsV1Alpha1Client, templateName, path, version, repo, icon string) {
+	// Fetch the existing template store
+	template, err := clientset.TemplateStore("cyclops").Get(templateName)
+	if err != nil {
+		fmt.Println("Failed to fetchtemplate store ", err)
```

Comment:
```suggestion
		log.Fatal("Failed to fetch template store:", err.Error())
```

Reply:
@petar-cvit I have a doubt that we need the user to know that we are unable to fetch right, logging only doesn't indicate the user that where they had missed ?

![image](https://github.com/user-attachments/assets/5ed06925-b05b-4684-85a5-fc4522c92697)

After updating I run with the wrong name of the template to check but I got only the above as a output which will mean , as a user I am unable to know what did I missed right?

Reply:
@Abiji-2020, you got this output because you invoked `update templates` instead `update template`. 

```
 ./cyctl update template some-template --rep
o newrepo
2024/10/10 11:48:22 Failed to fetch template store:templatestores.cyclops-ui.com "some-template" not found
```

You will need to add an alias to your command

Reply:
Here is the comment on where to add an alias https://github.com/cyclops-ui/cyclops/pull/607#discussion_r1795088401

Reply:
sorry for the confusion